app/main.py

```diff
@@ -0,0 +1,83 @@
+import sys
+
+from fastapi import Request, FastAPI, HTTPException
+
+from linebot.v3.webhook import WebhookParser
+from linebot.v3.messaging import (
+    AsyncApiClient,
+    AsyncMessagingApi,
+    Configuration,
+    ReplyMessageRequest,
+    TextMessage
+)
+from linebot.v3.exceptions import (
+    InvalidSignatureError
+)
+from linebot.v3.webhooks import (
+    MessageEvent,
+    TextMessageContent
+)
+import uvicorn
+from .config import Settings
+from .logger import Logger
+
+
+setting = Settings()
+logger = Logger().get()
+# get channel_secret and channel_access_token from your environment variable
+if setting.LINE_CHANNEL_SECRET is None:
+    print('Specify LINE_CHANNEL_SECRET as environment variable.')
+    sys.exit(1)
+if setting.LINE_CHANNEL_ACCESS_TOKEN is None:
+    print('Specify LINE_CHANNEL_ACCESS_TOKEN as environment variable.')
+    sys.exit(1)
+
+configuration = Configuration(
+    access_token=setting.LINE_CHANNEL_ACCESS_TOKEN
+)
+
+app = FastAPI()
+async_api_client = AsyncApiClient(configuration)
+line_bot_api = AsyncMessagingApi(async_api_client)
+# INFO: WebhookParser should take channel_secret
+parser = WebhookParser(channel_secret = setting.LINE_CHANNEL_SECRET)
+
+@app.get("/")
+async def root():
```

Comment:
Maybe remove async? Since in FastAPI using async without await may cause more time for execution.

Reply:
Nice suggestion! Already fixed.